src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -1195,9 +1195,13 @@ Copyright (c) .NET Foundation. All rights reserved.
           DestinationFiles="$(SdkOutputDirectory)RuntimeIdentifierGraph.json"
           SkipUnchangedFiles="true"/>
 
+    <Copy SourceFiles="$(NuGetPackageRoot)/microsoft.netcore.platforms/$(_NETCorePlatformsPackageVersion)/PortableRuntimeIdentifierGraph.json"
```

Comment:
For my understanding:
Is this copying from a package that is produced by the runtime repo build?
Did we not have to copy this previously because it came with the sdk repo build output?

Reply:
Exactly. The package is still produced in runtime. The portable RID graph was previously checked into the sdk directly. We want to remove that extra copy now and use the runtime package source of truth.